PULL_REQUEST_TEMPLATE

```diff
@@ -1,11 +1,9 @@
---- erase from here
-
-**Merge/Pull request title name template, please delete it before saving it**
-
+<!--
+Ensure that the title follows this template (or, just use the Asana task title):
```

Comment:
The title of the Asana task doesn't have to have the same structure than the PR (task_type is not in asana).

Reply:
Oh, I see. I've reverted this.